pycode/memilio-epidata/memilio/epidata/getPopulationData.py

```diff
@@ -136,7 +136,7 @@ def load_population_data(out_folder=dd.defaultDict['out_folder'],
    @param read_data False or True. Defines if data is read from file or downloaded. Default defined in defaultDict.
    @param no_raw True or False. Defines if unchanged raw data is written or not. Default defined in defaultDict.
    @param file_format File format which is used for writing the data. Default defined in defaultDict.
-   @return 3 Dataframes of migration, reg_key and zensus
+   @return Three dataframes of county_table, reg_key and zensus data.
```

Comment:
Line 123:
`"""! Load of counties, zensus and reg_key files` 
should be:
`"""! Load of county_table, zensus and reg_key files`.

Line 127:
`data for population per county [stored in "county_table"]`